### PR TITLE
Feature/frontend endpoint setup

### PR DESCRIPTION
**Pull request by SlimPortfolio**

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import Home from './components/Home';
 import Events from './components/Events';
 import Login from './components/Login';
 import { BrowserRouter, Route, Routes } from 'react-router-dom';
-
+import axios from 'axios';
```
Comment (StreetsJ) [inline client/src/App.js:7]: You can remove this import

File: server/app.js [hunk 2]
```diff
@@ -4,13 +4,10 @@ import {getData} from './database.js'
...
 
-// app.post("/events", async (req, res) => {
-//     const {title, description } = req.body
-//     const events = await getData()
-// })
-
 app.get("/", async (req, res) => {
     const events = await getData()
+    //responding to get request by allowing localhost:3000 to access API, and also sending events variable over
+    res.header("Access-Control-Allow-Origin","http://localhost:3000");
```
Comment (StreetsJ) [inline server/app.js:10]: You shouldn't need this line

Comment (SlimPortfolio) [reply server/app.js:10]: Oddly enough, when I was working on it before, it was giving me the CORS errors...after I deleted this, the app still functions as it should... Any idea why I would have seen issues in the first place? hmmm

Comment (SlimPortfolio) [reply server/app.js:10]: UPDATE: I tried adding an item to my DB, and then when I ran the program again, I got a CORS Error from the server. After I  added this info, it was able to work again.

File: client/src/App.js (hunk 1 above)
Comment (SlimPortfolio) [reply client/src/App.js:7]: removed!

File: server/app.js (hunk 2 above)
Comment (StreetsJ) [reply server/app.js:10]: Gotcha okay yeah I can look into that

**Pull request merged**